src/resources/invoices-resource.js

```diff
@@ -1,136 +1,126 @@
-            return apiHandler.post(`invoices/${id}/issue`, data);
-        },
-
-        reissue({id, data = {}}) {
-            return apiHandler.post(`invoices/${id}/reissue`, data);
-        },
-
-        abandon({id}) {
-            return apiHandler.post(`invoices/${id}/abandon`);
-        },
-
-        void({id}) {
-            return apiHandler.post(`invoices/${id}/void`);
-        },
-
-        getAllInvoiceItems({id, limit = null, offset = null, expand = null}) {
-            const params = {
-                limit,
-                offset,
-                expand,
-            };
-            return apiHandler.getAll(`invoices/${id}/items`, params);
-        },
-
-        createInvoiceItem({id, data}) {
-            return apiHandler.post(`invoices/${id}/items`, data);
-        },
-
-        getLeadSource({id}) {
-            return apiHandler.get(`invoices/${id}/lead-source`);
-        },
-
-        createLeadSource({id, data}) {
-            return apiHandler.put(`invoices/${id}/lead-source`, data);
-        },
-
-        updateLeadSource({id, data}) {
-            return apiHandler.put(`invoices/${id}/lead-source`, data);
-        },
-
-        deleteLeadSource({id}) {
-            return apiHandler.delete(`invoices/${id}/lead-source`);
-        },
-
-        getAllTimelineMessages({id= null, limit = null, offset = null, sort = null, filter = null, q = null} = {}) {
-            const params = {
-                limit,
-                offset,
-                sort,
-                filter,
-                q,
-            };
-            return apiHandler.getAll(`invoices/${id}/timeline`, params);
-        },
-
-        getTimelineMessage({id, messageId}) {
-            return apiHandler.get(`invoices/${id}/timeline/${messageId}`);
-        },
-
-        deleteTimelineMessage({id, messageId}) {
-            return apiHandler.delete(`invoices/${id}/timeline/${messageId}`);
-        },
-
-        createTimelineComment({id, data}) {
-            return apiHandler.create(`invoices/${id}/timeline`, '', data);
-        },
-
-        getAllTransactionAllocations({id, limit = null, offset = null}) {
-            const params = {
-                limit,
-                offset,
-            };
-
-            return apiHandler.getAll(`invoices/${id}/transaction-allocations`, params);
-        },
-        
-        applyTransaction({id, data}) {
-            return apiHandler.post(`invoices/${id}/transaction`, data);
-        },
-    };
-};
+  return {
+    getAll({
+      filter = null,
+      sort = null,
+      limit = null,
+      offset = null,
+      q = null,
+      expand = null,
+    } = {}) {
+      const params = {filter, sort, limit, offset, q, expand};
+      return apiHandler.getAll(`invoices`, params);
+    },
+    create({id = '', data, expand = null}) {
+      const params = {expand};
+      return apiHandler.create(`invoices/${id}`, id, data, params);
+    },
+    get({id, Accept, expand = null}) {
```

Comment:
What is `Accept` here?

Reply:
Good catch. There was a bug in the generator, that was including `header` parameters.